build.sbt

```diff
@@ -34,6 +34,7 @@ lazy val commonSettings = Seq(
 
 lazy val core = (project in file("core"))
   .dependsOn(storage)
+  .dependsOn(storageDynamodb % "test->compile") // delta-core tests depend on storageDynamodb
```

Comment:
Why not make `storageDynamodb`'s `test` depends on `core`'s `test`?

Reply:
ACK